csfieldguide/chapters/content/en/human-computer-interaction/sections/user-experience.md

```diff
@@ -0,0 +1,559 @@
+
+**UX Methods: Empathy Map, User Persona and User Stories**
+
+Now that we’ve established who our users are and what they want, it would be useful to create a *User Persona*.
+A *User Persona* is a character that is representative of your target user.
+To help us with creating a user persona, we will use an *Empathy Map* to give us a structure to work with.
+An *Empathy Map* defines four aspects of the persona: how they Think and Feel and what they Do and Say, and is drawn in quadrants.
+We’ve created a very basic one below to help us develop a persona for the target user of our sleep habits app:
+
+|                                                                                                                                          	| **User**                                                                                                                                                                                                                    	|
+|------------------------------------------------------------------------------------------------------------------------------------------	|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------	|
+| **Says**<br>Things the user says out loud in an interview or wrote in their survey.<br><br>*I am so tired.<br>I think I stayed up too late last night.* 	| **Thinks**<br>What do they think about? What are their needs, goals and motivations?<br><br>*I wish I could get more sleep.<br>Maybe I should go to bed earlier.<br>How many hours of sleep do I actually need?<br>Why am I always tired?* 	|
+| **Does**<br>What actions does the user do?<br><br>*Always hits “snooze” on the alarm.<br>Drinks coffee first thing in the morning.*                     	| **Feels**<br>Consider the tone they use in an interview and/or survey answers.<br><br>*Tired.<br>Annoyed.<br>Frustrated.*                                                                                                                	|
+
+Using our Empathy Map we are able to derive the target users needs and from here we created a persona representative of our target user.
+Meet Bevan:
+
+**Bevan, 22, Accounting Student at University**
+
+{image file-path="img/chapters/bevan.jpg" alt="Photo of Bevan for persona."}
+
+*__About__*
+
+*Bevan is a busy undergraduate student.
+He gets good grades and plays in a social soccer league.
+He is aware that he probably isn’t getting enough sleep, particularly as the exam period is coming up and he’s staying up late most nights to study.
+He knows that if he could improve the quality of the sleep that he is getting he would be more productive when studying and have more energy for this soccer games and training.*
+
+*__Goals__*
+
+*- Wake up feeling rested.*
+*- Be more productive during the day rather than feeling slow and sluggish.*
```

Comment:
Formatting error here and immediately below
![image](https://user-images.githubusercontent.com/25916475/58769910-91c0d800-85ff-11e9-9855-c6e7016886c1.png)

Reply:
I think it's because the `*` characters should be around the text of the bullet points, and not the dashes as well. The spacing seems correct.

Reply:
Markdown lists (ours anyway) need a space between items, it's been done elsewhere in the same file